reconcile/test/test_quay_mirror.py

```diff
@@ -78,3 +78,29 @@ def test_is_compare_tags_force_compare(self, mock_gql, mock_settings, mock_time)
             compare_tags=True,
         )
         assert qm.is_compare_tags
+
+
+@pytest.mark.parametrize(
+    "tags, tags_exclude, candidate, result",
+    [
+        (["^sha256-.+sig$", "^main-.+"], None, "main-755781cc", True),
+        (["^sha256-.+sig$", "^main-.+"], None, "sha256-8b5.sig", True),
+        (["^sha256-.+sig$", "^main-.+"], None, "1.2.3", False),
+        (None, ["^sha256-.+sig$", "^main-.+"], "main-755781cc", False),
+        (None, ["^sha256-.+sig$", "^main-.+"], "sha256-8b5.sig", False),
+        (
+            ["^sha256-.+sig$", "^main-.+"],
+            ["^sha256-.+sig$", "^main-.+"],
+            "main-755781cc",
+            True,
```

Comment:
nit: maybe a short comment that `tags` are indeed intended to take precedence over `tags_exclude`. I know this is mentioned in the schemas type description, but it confused me here a bit initially :sweat_smile:

Reply:
will do